src/Neo/SmartContract/ApplicationEngine.cs

```diff
@@ -65,8 +65,11 @@ public partial class ApplicationEngine : ExecutionEngine
 
         private static Dictionary<uint, InteropDescriptor> services;
         // Total amount of GAS spent to execute.
-        // In the unit of datoshi, 1 datoshi = 1e-8 GAS, 1 GAS = 1e8 datoshi
-        private readonly long _feeAmount;
+        // In the unit of pico-datoshi, 1 datoshi = 1e-8 GAS, 1 GAS = 1e8 datoshi
```

Comment:
pico itself is 10^{-12}, so maybe we need to refine this name,
Because than it would literally means 10^{-12}*10^{-8} GAS